### PR TITLE
design review fixes

### PR DESCRIPTION
**Pull request by scwambach**
## Description

Updated styles requested by design
### How to test

Add a program http://localhost:3000/mcpar and scrunch the screen down.

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

**Comment by ntsummers1** [comment]
Beat me too it on the font-size update there 😆, but looks great on my end!

File: services/ui-src/src/routes/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -386,6 +386,11 @@ const sx = {
     marginBottom: "1rem",
     fontSize: "4xl",
     fontWeight: "normal",
+    ".mobile &": {
+      fontSize: "xl",
+      lineHeight: "27.3px",
```
[the remark this answers is not in the document]
Comment (scwambach) [reply services/ui-src/src/routes/Dashboard/Dashboard.tsx:391]: Yeah, I thought that was odd number, but that's what she specified in the comment so I just went for it

[the remark this answers is not in the document]
Comment (braxex) [reply services/ui-src/src/routes/Dashboard/Dashboard.tsx]: Yeah that's weird. Let's just do 1.75rem and call it good.

**Review by braxex — DISMISSED** [review]
The only thing I see is that we still have a weird area between tablet and mobile where the dashboard gets cut off. What did design say about this?

**Pull request closed without merging**